agglayer-attach-cdk-params.yml

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
nit: we should fin a way to run the procedure without having to duplicate `params.yml`.